# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/xpconnect/tests/mochitest/test_bug916945.html
# Commit: b668ff9bcb38
# Full Hash: b668ff9bcb3887be19b7a7f8d9e9a51e5667b3b5
# Author: Paul Bone <pbone@mozilla.com>
# Date: 2019-09-10 15:48:08
# Regressor Bug: 1578628
# File Overlap Count: 1
# Description:
#   Bug 1578628 - Enable disabled test points r=bholley,kmag
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44945
# ==============================================================================

diff -r 14a24398b65b -r b668ff9bcb38 js/xpconnect/tests/mochitest/test_bug916945.html
--- a/js/xpconnect/tests/mochitest/test_bug916945.html	Tue Sep 10 05:42:25 2019 +0000
+++ b/js/xpconnect/tests/mochitest/test_bug916945.html	Tue Sep 10 07:04:54 2019 +0000
@@ -24,8 +24,7 @@
     ok('winA' in window, "same-origin named access works");
     ok(winA instanceof Window, "same-origin named access works");
     ok('winB' in window, "cross-origin named access works when iframe name matches");
-    // XXXbholley - flip me when Window moves to new bindings.
-    todo_is(winB instanceof Window, "cross-origin named access works when iframe name matches");
+    ok(winB instanceof Window, "cross-origin named access works when iframe name matches");
 
     // Setting the 'name' attribute should propagate to the docshell.
     var ifrB = document.getElementById('ifrB');
@@ -35,8 +34,7 @@
                    'attribute sets propagate to the docshell');
     });
     ok('foo' in window, "names are dynamic if updated via setAttribute");
-    // XXXbholley - flip me when Window moves to new bindings.
-    todo('foo' instanceof Window, "names are dynamic if updated via setAttribute");
+    ok(foo instanceof Window, "names are dynamic if updated via setAttribute");
 
     // Setting window.name on the subframe should not propagate to the attribute.
     await SpecialPowers.spawn(ifrB, [], () => {
